manifests/potato-facts.yml

```diff
@@ -30,7 +30,7 @@ spec:
     spec:
       containers:
 #      - image:  armory/potato-facts:v1.6.0 #v1.5.1 #v1.6.0 # v1.7.0
-      - image:  index.docker.io/armory/potatofacts:v2 #demoimages/bluegreen:v5 #v5, v4, v3
+      - image:  demoimages/bluegreen:latest #demoimages/bluegreen:v5 #v5, v4, v3
```

Comment:
Not sure why the image change is needed? I am guessing its because of the bug where the UI says pods are Ready before they actually are.